plus/src/test/java/org/apache/calcite/adapter/tpcds/TpcdsTest.java

```diff
@@ -367,15 +366,13 @@ protected void foo(CalciteAssert.AssertThat with, String tableName,
                 builder.equals(builder.field("CD_EDUCATION_STATUS"),
                     builder.literal("HIGH SCHOOL")),
                 builder.equals(builder.field("D_YEAR"), builder.literal(1998)),
-                builder.call(SqlStdOperatorTable.IN,
-                    builder.field("S_STATE"),
```

Comment:
This is a breaking change in a minor release.

Reply:
Hi, I stumbled upon this breaking change as well and unfortunately it is not categorized as breaking change in [Calcite release notes](https://calcite.apache.org/docs/history.html#v1-26-0).